### ISSUE TITLE
SCP problem

### ISSUE THREAD

**Issue by sh4tteredd** [eisaev/r2350#15]
Hi,
I'm trying to update my AC2350 (black, INT version) from mod6 to mod7, but anytime I time to use scp (transfer file from me to router and vice versa) I get this:
<img width="371" alt="Screenshot 2022-04-24 at 14 21 36" src="https://user-images.githubusercontent.com/55893559/164976273-6fa04f92-8483-4e79-8574-9d1f1f1a3342.png">

I'm on macOS, idk if this can matter.

**Comment by hartkopp** [comment]
AFAIK `scp` has different modes https://wiki.archlinux.org/title/SCP_and_SFTP

I did not have issues on Linux using `scp`.

Maybe you can switch the protocol handling on macOS ?!?

**Comment by sh4tteredd** [comment]
<img width="950" alt="Screenshot 2022-04-25 at 15 52 34" src="https://user-images.githubusercontent.com/55893559/165103160-09e05995-df6e-4e41-a166-4b11a7c9002c.png">

Just found out why in the Arch wiki (the link that @hartkopp sent) and it was -O option to use the legacy mode of file transfer over SSH.

In my case so the command has to be : `scp -O root@192.168.178.120:/tmp/ALL.backup ./`

it would be great if @eisaev can add this note for OpenSSH >= 9 in the readme

**Comment by hartkopp** [comment]
> Just found out why in the Arch wiki (the link that @hartkopp sent) and it was -O option to use the legacy mode of file transfer over SSH.

The Arch Linux URL was the first hit that pointed to the different SCP protocol modes ;-)
In fact I was using Ubuntu/Debian here.

> In my case so the command has to be : `scp -O root@192.168.178.120:/tmp/ALL.backup ./`
> 
> it would be great if @eisaev can add this note for OpenSSH >= 9 in the readme

You might just send a pull request with your suggested documentation update.

**Comment by sh4tteredd** [comment]
yeah, at the beginning I didn't care that much about the wiki differences about the 2 protocols, my bad. Thanks, btw I will open a pull request.
